includes/Checker/Default_Check_Repository.php

```diff
@@ -55,6 +55,7 @@ private function register_default_checks() {
 				'localhost'                  => new Checks\Localhost_Check(),
 				'no_unfiltered_uploads'      => new Checks\No_Unfiltered_Uploads_Check(),
 				'trademarks'                 => new Checks\Trademarks_Check(),
+				'plugin_header_readme'       => new Checks\Plugin_Header_Readme_Check(),
```

Comment:
Why new check registered? Instead of it can we introduce new method in `Plugin_Readme_Check`

Reply:
New check is needed becouse it compares readme and plugin header.

Reply:
We could potentially do that in the existing check too if desired

Reply:
I don't see a problem. I've created to use for checks that has to compare plugin header with the readme.

Reply:
If we combine the two classes there would be less redundancy and maintenance overhead